xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2006,7 +2006,11 @@ void COutput::InitCycle()
         ||  method == VS_INTERLACEMETHOD_RENDER_BOB)
     {
         if (method == VS_INTERLACEMETHOD_AUTO)
+#if defined(HAVE_VPP)
```

Comment:
if we want to consider every silly scenario that may come to user's minds, we need to introduce some thousands more of those ifdefs. Having this available in a build environment is a requirement now.